src/signals/create_source.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
```

Comment:
Maybe we can simply implement it in C by simply using `#include <signal.h>`, without shelling out the introspection. e.g.:
https://github.com/karelzak/util-linux/blob/917f53cf13c36d32c175f80f2074576595830573/lib/signames.c#L55-L132

Reply:
We could do that if we only support posix signals.

Reply:
@Snaipe Whats your opinion on this? Posix or the local signals?